source/assets/javascripts/app/_download.js

```diff
@@ -158,6 +160,10 @@ var showDownloadLinks = (function ($) {
         $("#download-revisions-template").html()
       );
 
+      // adding the docker organisation
+      latest_cloud_release.docker_org = settings.docker_org;
+      other_cloud_releases.docker_org = settings.docker_org;
```

Comment:
Instead of this, you should use `R.assoc`. See `addDisplayVersion` and `addReleaseDate` functions.

Reply:
Done.